src/tests/pox-tests.ts

```diff
@@ -141,4 +141,116 @@ describe('PoX tests', () => {
+      total: 1,
+    });
+
+    const cycle = await supertest(api.server).get(`/extended/v2/pox/cycles/6`);
+    expect(cycle.status).toBe(200);
+    expect(cycle.type).toBe('application/json');
+    expect(JSON.parse(cycle.text)).toStrictEqual({
+      block_height: 14,
+      cycle_number: 6,
+      index_block_hash: '0x8be3f0e3b746452309347a3f2deec2bba0b1d85642092c30262c9e07e10f3529',
+      total_signers: 3,
+      total_stacked_amount: '17501190000000000',
+      total_weight: 21,
+    });
+
+    const signers = await supertest(api.server).get(`/extended/v2/pox/cycles/6/signers`);
+    expect(signers.status).toBe(200);
+    expect(signers.type).toBe('application/json');
+    expect(JSON.parse(signers.text)).toStrictEqual({
+      limit: 100,
+      offset: 0,
+      results: [
+        {
+          signing_key: '0x028efa20fa5706567008ebaf48f7ae891342eeb944d96392f719c505c89f84ed8d',
+          stacked_amount: '7500510000000000',
+          stacked_amount_percent: 42.857142857142854,
+          weight: 9,
+          weight_percent: 42.857142857142854,
+        },
+        {
+          signing_key: '0x023f19d77c842b675bd8c858e9ac8b0ca2efa566f17accf8ef9ceb5a992dc67836',
+          stacked_amount: '5000340000000000',
+          stacked_amount_percent: 28.571428571428573,
+          weight: 6,
+          weight_percent: 28.57142857142857,
+        },
+        {
+          // steph doubled the weight of this signer
+          signing_key: '0x029fb154a570a1645af3dd43c3c668a979b59d21a46dd717fd799b13be3b2a0dc7',
+          stacked_amount: '5000340000000000',
+          stacked_amount_percent: 28.571428571428573,
+          weight: 6,
+          weight_percent: 28.57142857142857,
+        },
+      ],
+      total: 3,
+    });
+
+    const signer = await supertest(api.server).get(
+      `/extended/v2/pox/cycles/6/signers/0x029fb154a570a1645af3dd43c3c668a979b59d21a46dd717fd799b13be3b2a0dc7`
+    );
+    expect(signer.status).toBe(200);
+    expect(signer.type).toBe('application/json');
+    expect(JSON.parse(signer.text)).toStrictEqual({
+      signing_key: '0x029fb154a570a1645af3dd43c3c668a979b59d21a46dd717fd799b13be3b2a0dc7',
+      stacked_amount: '5000340000000000',
+      stacked_amount_percent: 28.571428571428573,
+      weight: 6,
+      weight_percent: 28.57142857142857,
+    });
+
+    const stackers = await supertest(api.server).get(
+      `/extended/v2/pox/cycles/6/signers/0x029fb154a570a1645af3dd43c3c668a979b59d21a46dd717fd799b13be3b2a0dc7/stackers`
+    );
+    expect(stackers.status).toBe(200);
+    expect(stackers.type).toBe('application/json');
+    expect(JSON.parse(stackers.text)).toStrictEqual({
+      limit: 100,
+      offset: 0,
+      results: [
+        {
+          pox_address: '1NQAp2ecV2zUQfz7y2hLvJTU5scnFTSACM',
+          stacked_amount: '2500170000000000',
+          stacker_address: 'ST3NBRSFKX28FQ2ZJ1MAKX58HKHSDGNV5N7R21XCP', // signer
+        },
+        {
+          pox_address: 'mhYeZXrSEuyf2wbJ14qZ2apG7ofMLDj9Ss',
```

Comment:
This pox_address is different in the test run -- not sure if an encoding error somewhere

Reply:
It's `132hGUmTRtYQFq7gHVsBCfbwFp4eNsQvBi`, so I'm assuming either core or the API is encoding as mainnet, instead of testnet

Reply:
Hmm yes we get the bitcoin address directly from the node. Perhaps the `pox-addr` clarity value tuple generated in the tests used a mainnet version byte? Otherwise it does sound like a potential issue in core. Either way, I don't think it's an issue in the API codebase, so we can merge this.

Reply:
Found it. The TSV import gets the chainId from ENV and uses that to parse the pox_addr_raw